### ISSUE TITLE
Allow configuring to default to a particular instance

### ISSUE THREAD

**Issue by NoraCodes** [cheeaun/phanpy#847]
**Is your feature request related to a problem? Please describe.**

I host several GoToSocial servers. Since GTS doesn't include a frontend, I host Phanpy at `/client` and put a link there in my instance description. Users then have to hit log in, type in the URL of the instance, and log in. This is not great UX.

**Describe the solution you'd like**
It would be much more convenient if, upon clicking through, users were presented with a button to log in to that instance. Even better would be if I could link to a route that immediately redirected to the login page for my instance, so it would function as a true login button.

In other words, rather than:

Front page -> Phanpy front page -> Enter instance name -> Log in form -> Phanpy interface

I'd like to be able to allow users to do

Front page -> Log in form -> Phanpy interface

**Comment by cheeaun** [comment]
@NoraCodes are you using the "Easy way" or "Custom-build way" for [self-hosting](https://github.com/cheeaun/phanpy?tab=readme-ov-file#self-hosting)?
